.github/workflows/python-app.yml

```
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python application
```

Maybe a more descriptive name? Ie. `Indigent Defense Stats`

```suggestion
name: Indigent Defense Stats
```